src/modules/account/types.ts

```diff
@@ -0,0 +1,38 @@
+import { TSMap } from 'typescript-map'
+import { TransferState } from './RadixTransferAccountSystem'
+import { TokenDefinitionState } from './RadixTokenDefinitionAccountSystem'
+
+export type LedgerState = {
+    [key: string]: AccountState
```

Comment:
Is the key an address? Why not all it that then?

Reply:
Lol, for some reason I thought you had to put the name "key" there when specifying the type like this. Learned something new :D